purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -174,7 +174,6 @@ class Purchases internal constructor(
             log(LogIntent.WARNING, AUTO_SYNC_PURCHASES_DISABLED)
         }
 
-        offlineEntitlementsManager.updateProductEntitlementMappingCacheIfStale()
```

Comment:
This one wasn't really needed since `onAppForeground` is also executed on initial SDK configuration, even if it's AFTER the app is open. We also request offerings from there, which is much more important.